src/main/java/core/basesyntax/service/impl/FileReaderCvs.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.IFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderCvs implements IFileReader {
+
+    @Override
+    public List<String> read(String path) {
+        List<String> lineList;
+        try {
+            lineList = Files.readAllLines(Path.of(path));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find file by path: " + path, e);
```

Comment:
its not guaranteed that exception will be related only to not finding file
```suggestion
            throw new RuntimeException("Can't read file by path: " + path, e);
```